### PR TITLE
Fix unknown instance type

### PR DESCRIPTION
**Pull request by nuwang**
This PR switches to the `describe_instance_types` API to populate `instance_types.json`, instead of using `www.ec2instances.info`. This fixes an issue where the `describe_instance_type_offerings` API call returns instances which are not listed in `instance_types.json`

ping @spulec @bblommers

### CODE REVIEW

**Review by bblommers — CHANGES_REQUESTED** [review]
Thanks for the PR, @nuwang! Makes sense, to get the data directly from AWS.
Please see my comments - happy to merge once that's addressed

File: scripts/get_instance_info.py [hunk 1]
```diff
@@ -3,157 +3,44 @@
...
-    print("Finding data in table")
-    instances = data_table.find("tbody").find_all("tr")
+    print("Getting InstanceTypes from all regions")
+    regions = []
+    regions.extend(Session().get_available_regions("ec2"))
+    regions.extend(Session().get_available_regions("ec2", partition_name="aws-us-gov"))
+    regions.extend(Session().get_available_regions("ec2", partition_name="aws-cn"))
+    print("Found " + str(len(regions)) + " regions")
+
+    for region in ["us-east-1"]:
```
[the remark this answers is not in the document]
Comment (nuwang) [reply scripts/get_instance_info.py:22]: Thanks for the catch! Fixed.

[the remark this answers is not in the document]
Comment (bblommers) [reply scripts/get_instance_info.py:22]: Doesn't use the regions-variable yet, @nuwang :)

[the remark this answers is not in the document]
Comment (nuwang) [reply scripts/get_instance_info.py]: @bblommers Oops! I had missed this one which I used while testing. Sorry about that.

**Comment by nuwang** [comment]
One sec, am updating instance types from all regions.

**Comment by nuwang** [comment]
All done.

**Review by bblommers — APPROVED** [review]
LGTM - thanks again @nuwang!

**Comment by bblommers** [comment]
This is now part of moto >= 1.3.17.dev283

**Comment by nuwang** [comment]
Thanks for the review and merge!

**Pull request merged**